### ISSUE TITLE
Details for Export command

### ISSUE THREAD

**Issue by gmsshadow** [playbymail/Far-Horizons#6]
Can we add to the documentation that the proper usage for the export commands are:

FH_SEED=$RANDOM ../build/fh export json galaxy
FH_SEED=$RANDOM ../build/fh export json species
FH_SEED=$RANDOM ../build/fh export json planets
FH_SEED=$RANDOM ../build/fh export json stars

**Comment by mdhender** [comment]
The command line interface for exports was changed when I updated the import code. Now all of the data is exported so that the files are always in sync with each other. The command line is simpler now, its just `fh export json.`

The README's been updated. This should be ready to close.

[Exporting data to JSON](https://github.com/playbymail/Far-Horizons/tree/main#exporting-data-to-json)
The fh export json command exports game data as JSON formatted data and saves the result to text files (instead of the normal binary data files).

The output file names are hard-coded:

galaxy.json
stars.json
planets.json
species.NNN.json
species.NNN.planets.json
species.NNN.ships.json
locations.json
transactions.json
Each species creates a set of three files, one for the species data, one for the planets the species has named, and one for all the ships the species controls.

[Importing data from JSON](https://github.com/playbymail/Far-Horizons/tree/main#importing-data-from-json)
The fh import json command imports game data from JSON formatted data and saves the result to the binary data files.

It expects the same file names that the fh export json creates.

There are some data quality checks on the import, but they need much more work.

Note that if the import does detect issues with the input, it will abend instead of overwriting the binary data files.
